### ISSUE TITLE
Add parameter to disable brotli compression on architect

### ISSUE THREAD

**Issue by Zaxuhe** [architect/architect#1341]
**Is your feature request related to a problem? Please describe.**
We have several users that do not support brotli compression (older devices/esp32 hardware), currently we are blocked from going 10.3.xxx+

**Describe the solution you'd like**
As part of commit https://github.com/architect/deploy/commit/1a53685412dd0a86de6bd9b5b20bd3fc9699355a brotli compression was added, please add a parameter to remove it https://github.com/architect/architect/blob/8eb576e0438cd298bad29630c27401585aa4d033/changelog.md

**Describe alternatives you've considered**
Currently we are blocked until such a parameter is added

**Additional context or notes**
Brotli is an aggressive compression algorithm that does not work on all hardwares due to CPU or size requirements.

**Comment by ryanblock** [comment]
Hmm, should all compression be disabled, or would gzip work?

**Comment by anthonyringoet** [comment]
@ryanblock Implementation-wise, would a new `compression` parameter (default: true) on `static` pragma be the way to go?

**Comment by ryanblock** [comment]
Hmm, perhaps! I'm not sure I'd agree that one would necessarily expect compression via CF, but that's certainly still a reasonable assumption.

Is there more info about "Android webviews" (I assume that refers to system web views?) not [supporting brotli](https://caniuse.com/?search=brotli)? That's pretty shocking, brotli is [Google's, after all](https://github.com/google/brotli).

Thanks!

**Comment by ryanblock** [comment]
Ok, just discussed at our checkin today and I'm going to roll this back a bit, and turn this feature into an opt-in.

Still debating what to do about the brotli compression we added to [`@architect/functions`](https://github.com/architect/functions), but since it's conditional on `accept-encoding` including `br`, I think it's probably safe to keep. See: https://github.com/architect/functions/blob/main/src/http/_res-fmt.js#L90

**Comment by ryanblock** [comment]
Ok, resolved here: https://github.com/architect/inventory/pull/41/commits/bcbe81184972a3d9c6378b294c312f964e64c67f + https://github.com/architect/deploy/pull/356/commits/179b42e87c02358ced19c1f99c77799676e45e14

tl;dr of changes: compression of static assets is turned off by default, `@static compression` of `true` or `br` enables brotli, or `gzip` enables gzip. One thing to note: we do not check compression of assets during upload, so if you change your setting, it won't be affected until the next asset content change, or prune, or other means of forcing asset publishing.

Now available in Arc 10.4 RC 1, should be released to production this week!
